test/suite/test_backup01.py

```diff
@@ -64,7 +64,7 @@ def populate(self, skiplsm):
     # Compare the original and backed-up files using the wt dump command.
     def compare(self, uri):
         self.runWt(['dump', uri], outfilename='orig')
-        self.runWt(['-h', self.dir, 'dump', uri], outfilename='backup')
+        self.runWt(['-h', self.dir, '-R', 'dump', uri], outfilename='backup')
```

Comment:
Please add a comment here (to all the tests needed) about why this is now necessary.